superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FilterConfigurePane.tsx

```diff
@@ -63,7 +63,14 @@ const FilterConfigurePane: React.FC<Props> = ({
   filters,
   removedFilters,
 }) => {
-  const active = filters.filter(id => id === currentFilterId)[0];
+  const getStyle: (id: string) => CSSProperties = useCallback(
```

Comment:
Do we have this pattern anywhere else? I believe we use Emotion components to handle these cases and pass props to them